jaeger-b3/src/test/java/com/uber/jaeger/propagation/b3/B3TextMapCodecTest.java

```diff
@@ -26,6 +26,8 @@
 import com.github.kristofa.brave.http.HttpServerRequest;
 import com.github.kristofa.brave.http.HttpServerRequestAdapter;
 import com.uber.jaeger.SpanContext;
+import com.uber.jaeger.propagation.HexCodec;
```

Comment:
it seems a bunch of tests in this file should be moved into core/propagation. This would also allow removing the dependency on the HexCodec instead of making it a public class

Reply:
Agree - actually just included a similar test in core/propagation, so will just move it instead.